src/main/java/com/buglife/sdk/TextInputField.java

```diff
@@ -62,11 +61,12 @@ public void setMultiline(boolean multiline) {
      * Returns the system-provided summary field (i.e. "What happened?").
      * When there are no custom input fields configured,
      * the bug reporter UI shows the summary field by default.
+     * @param title The summary field title
      */
-    public static TextInputField summaryInputField() {
+    public static TextInputField summaryInputField(String title) {
```

Comment:
This is a public API change which we should avoid; I'll append a commit that fixes this. Thanks!